app/components/participant-creator/component.js

```diff
@@ -36,21 +38,27 @@ export default Ember.Component.extend({
         }
         return ret;
     },
-    generatedParticipants: Ember.computed('batchSize', 'tag', function() {
+    exampleId: Ember.computed('tag', function() {
         var tag = this.get('tag');
-        var batchSize = Math.min(parseInt(this.get('batchSize')) || 0, 10);
-        return this._generate(batchSize, tag);
+        return `12345${tag ? `-${tag}` : ''}`;
     }),
 
     createdAccountsCSV: Ember.computed('createdAccounts', function() {
-        var keys = ['id'].concat(this.get('extra').map(e => `extra.${e.key}`));
+        var keys = ['id', 'extra.studyId'].concat(this.get('extra').map(e => `extra.${e.key}`));
         return keys.join(',') + '\n' + this.get('createdAccounts').map((a) => {
             var props = a.getProperties(keys);
             return keys.map(k => props[k]).join(',');
         }).join('\n');
     }),
     actions: {
         createParticipants() {
+            var studyId = this.get('studyId');
```

Comment:
See note about form validation below- the create event should be allowed to fire at all if the form is invalid.